### PR TITLE
st2[1386][IMP] stock_outgoing_shipment_report

### PR DESCRIPTION
**Pull request by kanda999**
[1386](https://www.quartile.co/web#id=1386&action=771&model=project.task&view_type=form&menu_id=505)

"lot_num" have no limit
add "stock_name"

### CODE REVIEW

File: stock_outgoing_shipment_report/models/stock_outgoing_shipment_report.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "product_name": ["Char", 32],
     "case_qty": ["Float", 5],
     "separate_qty": ["Float", 5],
-    "lot_num": ["Char", 10],
+    #"lot_num": ["Char", 10],
```
Comment (yostashiro) [inline stock_outgoing_shipment_report/models/stock_outgoing_shipment_report.py]: Remove this line.  No need to leave it as a comment.

**Review by yostashiro — APPROVED** [review]
Please take care of the commented part and also bump the module version (minor).

Go ahead and merge (squash and merge) once above is done.

File: stock_outgoing_shipment_report/models/stock_outgoing_shipment_report.py (hunk 1 above)
Comment (yostashiro) [reply stock_outgoing_shipment_report/models/stock_outgoing_shipment_report.py]: > Remove this line. No need to leave it as a comment.

Make sure that the pre-commit error disappears after this.

Comment (kanda999) [reply stock_outgoing_shipment_report/models/stock_outgoing_shipment_report.py]: ok I'll try it

**Pull request merged**